### PR TITLE
Release notes for 8.0.15-6

### PR DESCRIPTION
**Pull request by hrvojem**

### CODE REVIEW

File: doc/source/myrocks/variables.rst
Comment (laurynas-biveinis) [inline doc/source/myrocks/variables.rst:969]: George should review this, but IMHO "this is to be used by backup tools" or similar has to added, just like the other variables has it

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst]: Instead of copy pasting var descriptions, I'd rather say "The server part of MyRocks cross-engine consistent physical backups has been implemented by introducing two new vars X and Y"

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst]:
```suggestion
  :table:`performance_schema.log_status` table.
```

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst]: This needs docs outside RNs too

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/myrocks/variables.rst
Comment (george-lorch) [reply doc/source/myrocks/variables.rst:969]: Agree with @laurynas-biveinis we need a warning similar to something like "These variables are only to be used by automated backup tools.  Prolonged use or other misuse can have serious side effects to the server instance." or some other more proper English ;)

File: doc/source/release-notes/Percona-Server-8.0.15-6.rst
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-8.0.15-6.rst]: Is this supposed to be a part of this release?  We were supposed to be doing the MyRocks backup enabling only.  I am fine if this is intentional and has come at the direction of Tyler, Peter S. Laurynas or Sveta, but if it is accidental, then we should probably remove it from the release.  I have said my peace, do with it what you think best.

**Review by george-lorch — APPROVED** [review]
Approved pending any changes that may come from my comments.

**Comment by satya-bodapati** [comment]
@hrvojem What about bug fixes PS-5562 PS-5508 etc? wouldn't they make it to 8.0.15-6 release?

**Pull request merged**
